_replication-plugin/settings.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Settings
+nav_order: 40
+---
+
+# Replication settings
+
+The replication plugin adds several settings to the standard OpenSearch cluster settings.
+The settings are dynamic, so you can change the default behavior of the plugin without restarting your cluster.
+You can mark settings as `persistent` or `transient`.
+
+For example, to update the retention period of the result index:
```

Comment:
This request does not seem to be related to retention period... am I missing something? What is retention period in this context, anyway? The length of time before the follower index goes solo?

Reply:
Oops, I'm not sure what I was doing here but I think I intended to change this.